randovania/data/json_data/prime1.txt

```diff
@@ -2059,10 +2149,20 @@ Asset id: 3466748899
 > Dock to Frigate Crash Site; Heals? False
   > Frigate Crash Site/Dock to Overgrown Cavern
       Trivial
+  > Overgrown Cavern/Pickup (Missile Expansion)
+      Trivial
 
 > Dock to Transport Tunnel C; Heals? False
   > Transport Tunnel C/Dock to Overgrown Cavern
       Trivial
+  > Overgrown Cavern/Pickup (Missile Expansion)
+      Morph Ball
+
+> Pickup (Missile Expansion); Heals? False
+  > Overgrown Cavern/Dock to Frigate Crash Site
+      Trivial
+  > Overgrown Cavern/Dock to Transport Tunnel C
+      Morph Ball
 
 ----------------
 Transport to Chozo Ruins West
```

Comment:
Add another node to this room called "Elevator to Transport to Chozo Ruins West" with the requirement Scan Visor.

Reply:
I had planned to do this after getting the logic done. I will do this tomorrow aswell